openapi/components/schemas/RevenueEntry.yaml

```
@@ -0,0 +1,47 @@
+type: object
+readOnly: true
+properties:
+  customerId:
+    description: Customer's ID.
```

suggestion: It seems like we are repeating the variable name in the description

If we are describing object IDs, I think we should use this description
```suggestion
    description: Unique object identifier.
```
Or, if it's not a object:
```suggestion
    description: Unique customer identifier.
```

or

---

```suggestion
    description: Identifier of the customer which generated the revenue.
```

---

It's not really unique, @noon-dawg